In [5]:
### Load in necessary packages first
library(tidyverse)
library(repr)
library(tidymodels)
options(repr.matrix.max.rows = 6)

In [9]:
set.seed(1000) # Run this cell for reproducible results, DO NOT DELETE

# Project Planning Report

## Data Description

In this project I will be working with mainly the players.csv dataset. Below I have started introductory exploration of the dataset by reading it into the notebook, and having a quick preview of what it looks like in R.

In [11]:
playerdata <- read_csv("data/players.csv")
playerdata

Rows: 196 Columns: 7
── Column specification ────────────────────────────────────────────────────────
Delimiter: ","
chr (4): experience, hashedEmail, name, gender
dbl (2): played_hours, Age
lgl (1): subscribe

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.


experience subscribe
1   Pro         TRUE    
2   Veteran     TRUE    
3   Veteran    FALSE    
⋮   ⋮          ⋮        
194 Amateur    FALSE    
195 Amateur    FALSE    
196 Pro         TRUE    
    hashedEmail                                                     
1   f6daba428a5e19a3d47574858c13550499be23603422e6a0ee9728f8b53e192d
2   f3c813577c458ba0dfef80996f8f32c93b6e8af1fa939732842f2312358a88e9
3   b674dd7ee0d24096d1c019615ce4d12b20fcbff12d79d3c5a9d2118eb7ccbb28
⋮   ⋮                                                               
194 d572f391d452b76ea2d7e5e53a3d38bfd7499c7399db299bd4fedb06a46ad5bb
195 f19e136ddde68f365afc860c725ccff54307dedd13968e896a9f890c40aea436
196 d9473710057f7d42f36570f0be83817a4eea614029ff90cf50d8889cdd729d11
    played_hours name      gender            Age
1   30.3         Morgan    Male               9 
2    3.8         Christian Male              17 
3    0.0         Blake     Male              17 
⋮   ⋮            ⋮         ⋮                 ⋮  
194 0.0          Dylan     Prefer not to say 57 
195 2.3          Harlow    Male              17 
196 0.2          Ahmed     Other             NA

In the player.csv dataset there are 196 observations (rows) and 7 variables. The variables are as follows:
1. experience
    - Character variable with 5 different categories: Pro, Veteran, Amateur, Regular, Beginner

2. subscribe
    - Logical type variable denoting if a player is subscribed to game related newsletter
    
3. hashedEmail
    - Character variable that gives a player's hashed email address
    
4. played_hours
    - Double variable (integers with decimals) that reports how many hours played on the server by a player
    
5. name
    - Character variable giving a player's real name
    
6. gender
    - Character variable that gives the gender reported by a player
    
7. age
    - Double variable that gives the age reported by a player

**include summary statistics here**

Majority of the data was collected from players self reporting when initially signing up for the server except for played_hours which was collected via the server recording play sessions of the players.